docs/jemichael.brittain.md

```diff
@@ -0,0 +1,5 @@
+# GitHub jemichael.brittain
+
+- **Name:** Michael Brittain
+- **Occupation:** Food junkie
+- **Location** London
```

Comment:
What if you were more specific here?  Are you talking about the London in the United Kingdom? :gb: